modules/genomic_browser/php/models/cnvdto.class.inc

```diff
@@ -239,14 +239,4 @@ class CnvDTO implements \LORIS\Data\DataInstance
             'Platform'          => $this->_Platform,
         ];
     }
-    /**
-     * Returns the CenterID for this row, for filters such as
-     * \LORIS\Data\Filters\UserSiteMatch to match again.
```

Comment:
UserSiteMatch doesn't do anything with a function named `getPSC` and if it did, this is referencing a property that doesn't exist..